org.metaborg.spoofax.meta.core/src/main/resources/org/metaborg/spoofax/meta/core/generator/general/langspec/metaborg.yaml

```diff
@@ -14,10 +14,6 @@ dependencies:
 {{#analysisNabl2}}  - org.metaborg:org.metaborg.meta.nabl2.shared:${metaborgVersion}
   - org.metaborg:org.metaborg.meta.nabl2.runtime:${metaborgVersion}
 {{/analysisNabl2}}
-pardonedLanguages:
```

Comment:
At least for ESV and SDF3, I never pardon them, and do not have problems with that. I have the impression that the false positives happen in few projects only, and I miss actual errors when I don't realize they are pardoned. I'm not sure about Stratego actually, because I usually don't have it as an explicit dependency. Maybe these false positives are more common than I thought, then I'll put it back.